microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -247,14 +246,33 @@ def verify_gpu_cuda_with_pytorch(
         _install_driver(node, log_path, log)
         _check_driver_installed(node, log)
 
-        _install_cudnn(node)
+        # pytorch and CUDA needs 4GB space to download and install
+        free_partition = node.find_partition_with_freespace(5)
+        work_path = free_partition + "/python_packages"
```

Comment:
The directory logic should be moved into pip tool. So just pass the installation path in, and then create directories inside.

Reply:
changed